### PR TITLE
Add params to REST API table description, fix first column width

### PR DESCRIPTION
**Pull request by froggleston**
The previous PR #11102 has issues with the table rendering.

This PR fixes the first column width to make all endpoint URLs on a single line, and moves the parameters lists into the description column for readability.

{0580285C-7136-42DD-AC21-3EEAEA8DD994}

### CODE REVIEW

File: docs/stylesheets/ft.extra.css [hunk 1]
```diff
@@ -15,3 +15,7 @@
 .md-version__list {
     font-weight: 500 !important;
 }
+
+#available-endpoints + p + .md-typeset__scrollwrap .md-typeset__table th:first-of-type {
```
Comment (xmatthias) [inline docs/stylesheets/ft.extra.css]: This looks VERY fragile

simply adding another paragraph will break this completely ... is it possible to have a wildcard between the `+` characters instead?

Comment (xmatthias) [reply docs/stylesheets/ft.extra.css]:
```suggestion
#available-endpoints ~ .md-typeset__scrollwrap .md-typeset__table th:first-of-type {
```

this seems to work - but can you try this on your side, too?

Comment (froggleston) [reply docs/stylesheets/ft.extra.css]: Oh nice alternative - yeah I felt that was flaky but didn't have a nicer suggestion. Thanks!

Comment (xmatthias) [reply docs/stylesheets/ft.extra.css]: msdn docs for the rescue :rofl:

**Review by xmatthias — APPROVED** [review]
Great! 
LGTM  :+1:

**Pull request merged**